samples/mcuboot/firmware_loader_entrance/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  name: Firmware loader entrance
+  description: Demo application for entering the firmware loader application
+common:
+  sysbuild: true
+  build_only: true
```

Comment:
please remove build only to allow executing this sample in CI regression

Reply:
So test will not be complete (without remote device), but we can at least run that sample on device (check if sample is buildable, is up on device and if advertisement is started). I will open a PR for that.